build.py

```diff
@@ -35,7 +36,7 @@
 
 use_xcpretty = True
 
-class tclr:
+class colorValues:
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class "colorValues" to match the regular expression ^[A-Z_][a-zA-Z0-9]+$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS101)